examples/test_performance/make_data.py

```diff
@@ -0,0 +1,47 @@
+#This script creates the data necessary to make a graph of how
+#the latency and throughput of an NF chain changes based on length
+#outputs 3 numbers to 'data.txt': [#nfs, TX average, and latency average]
+
+num_nfs = input("Enter the number of nfs: ")
```

Comment:
in general since we want to automate these kinds of things it is better to make this a command line argument instead of user input

Reply:
The idea was to have the script input the # of NFs using the << or <<< command (I forgot which one works)

Reply:
Yeah that works but can get confusing if you have multiple inputs you want to define. 

It will be simpler to allow command line arguments. Here is a long tutorial explaining the principles https://realpython.com/python-command-line-arguments/
And here is a simpler one https://www.geeksforgeeks.org/command-line-arguments-in-python/